go.mod

```diff
@@ -165,4 +161,8 @@ require (
 // Required until we bump Cloud Credential Operator in Cloud Resource Operator - https://issues.redhat.com/browse/MGDAPI-4892
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20210831091943-07e756545ac1
 
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.7.17
+// Replaces for "image" and "libcontainer" are required to address CVEs Introduced through github.com/operator-framework/operator-registry@v1.36.0
+// Reminder: update the operator-registry and operator-lifecycle-manager on move to Go 1.21 and Go 1.22
+replace github.com/containers/image/v5 => github.com/containers/image/v5 v5.30.1
+
+replace github.com/opencontainers/runc/libcontainer => github.com/opencontainers/runc/libcontainer v1.1.12
```

Comment:
We need to remove it after bumping CCO dep in CRO and pulling new CRO in this PR.
It should be better then. Just leaving a note here to not forget about it

Reply:
Removed libcontainer replace (commented), but the CVE still appears. Looks like bump of CCO version didn't help. So will return this "replace" back in next commit. This is the CVE https://app.snyk.io/org/application-services-red-hat-openshift-api-management/project/c4a08403-992b-4c60-bdc6-84875659f44e/pr-check/20d2ffa8-aecb-4ec0-b231-ade2c80bf8ff . I'd like suggest investigate it in task https://issues.redhat.com/browse/MGDAPI-6350

Reply:
With recent changes it looks good, thanks.